Source/WebKit/DerivedSources.make

```diff
@@ -552,3 +561,27 @@ all : GeneratedSerializers.h GeneratedSerializers.cpp GeneratedSerializers.mm Se
 
 GeneratedSerializers.h GeneratedSerializers.cpp GeneratedSerializers.mm SerializedTypeInfo.cpp : $(WebKit2)/Scripts/generate-serializers.py $(SERIALIZATION_DESCRIPTION_FILES) $(WebKit2)/DerivedSources.make
 	$(PYTHON) $(WebKit2)/Scripts/generate-serializers.py $(WebKit2)/ $(SERIALIZATION_DESCRIPTION_FILES)
+
+EXTENSIONS_DIR = $(WebKit2)/WebProcess/Extensions
+EXTENSIONS_SCRIPTS_DIR = $(EXTENSIONS_DIR)/Bindings/Scripts
+EXTENSIONS_INTERFACES_DIR = $(EXTENSIONS_DIR)/Interfaces
+IDL_ATTRIBUTES_FILE = $(EXTENSIONS_SCRIPTS_DIR)/IDLAttributes.json
+
+BINDINGS_SCRIPTS = \
+    $(WebCoreScripts)/generate-bindings.pl \
+    $(WebCoreScripts)/IDLParser.pm \
+    $(WebCoreScripts)/CodeGenerator.pm \
+    $(EXTENSIONS_SCRIPTS_DIR)/CodeGeneratorExtensions.pm \
```

Comment:
I think this would be slightly more readable as ExtensionCodeGenerator, but if other code generators use this pattern (or even if it's a pain to fix), I don't feel strongly.

Reply:
Yeah this is the naming pattern for all the other generators. I don't think it technically matters, but I kept it.